mock/mock.spec

```diff
@@ -63,9 +63,18 @@ BuildRequires: python%{python3_pkgversion}-devel
 %if %{with lint}
 BuildRequires: python%{python3_pkgversion}-pylint
 %endif
-Requires: dnf
+
+%if 0%{?fedora} >= 38
```

Comment:
I would not change this for F38. Lets change it to > 38

Reply:
Hmm, ignore me, this is is plain suggest. That is fine, and it will automagically start working in F39+